### PR TITLE
O3-2895: Fix Layout of Patient Search Patient Banner

### PR DESCRIPTION
**Pull request by mogoodrich**
@denniskigen  I still feel I'm in the "knowing just enough to be dangerous" stage of Carbon and css :)  but this should hopefully fix the layout issues you mentioned:

Side note: for this (and others) I'm not testing a tablet layout in parallel, which I guess it something we maybe should start doing?

## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

### CODE REVIEW

**Review by mseaton — COMMENTED** [review]
I really have no insight.  Looks ok, but defer to @denniskigen / @ibacher or anyone else who has worked on these files and / or originally coded the CSS that is being replaced.

File: packages/esm-appointments-app/src/form/appointments-form.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@ $openmrs-background-grey: #ededed;
 
 :global(.omrs-breakpoint-lt-desktop) .formWrapper {
   background-color: $openmrs-background-grey;
+  margin-top: 3rem;
```
[the remark this answers is not in the document]
Comment (ibacher) [reply packages/esm-appointments-app/src/form/appointments-form.scss:15]: I think this is a pattern we've been using in a bunch of places, so it's probably fine. The breakpoints are defined [here](https://github.com/openmrs/openmrs-esm-core/blob/8a00793309f5e5b2143f98121f50075871d82761/packages/framework/esm-styleguide/src/breakpoints/index.ts#L12), but yeah, this one means "smaller than a desktop". Not my favourite thing.

For spacing, I think it's generally preferable to use Carbon's spacing utilities (`spacing.$spacing-09` in this case, though it ends up being `3rem` either way; this just—hopefully—will allow us to make more global styling decisions in the future).

**Comment by ibacher** [comment]
> I'm not testing a tablet layout in parallel, which I guess it something we maybe should start doing?

Yeah, it's a best practice to make sure things don't look awful.

File: packages/esm-patient-search-app/src/ui-components/overlay/overlay.scss [hunk 2]
```diff
@@ -51,15 +51,21 @@
...
+  vertical-align: center;
 }
 
-.closeButton {
-  background-color: $ui-02;
-  margin-right: spacing.$spacing-05;
-
+.closeButton,
+.closeButton:hover {
+  background-color: #fff;
```
Comment (ibacher) [inline packages/esm-patient-search-app/src/ui-components/overlay/overlay.scss:66]: Should this just be `white`? It's more consistent with the SVG fill.

**Review by ibacher — COMMENTED** [review]
LGTM, though I don't really like CSS much.

**Review by ibacher — APPROVED** [review]

File: packages/esm-appointments-app/src/form/appointments-form.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (mogoodrich) [reply packages/esm-appointments-app/src/form/appointments-form.scss:15]: Cool, great, thanks @ibacher  I was hoping there were some global stylings we could use for this, I switched this.

File: packages/esm-patient-search-app/src/ui-components/overlay/overlay.scss (hunk 2 above)
Comment (mogoodrich) [reply packages/esm-patient-search-app/src/ui-components/overlay/overlay.scss:66]: Just copying this from somewhere else, but that makes sense, will switch.

**Review by mogoodrich — COMMENTED** [review]
Thanks @ibacher , made the changes!

**Pull request merged**